mingw-w64-curl/PKGBUILD

```
@@ -1,27 +1,27 @@
 # Maintainer: Alexey Pavlov <alexpux@gmail.com>
 
-_variant=-openssl
-#_variant=-winssl
-#_variant=-gnutls
-
-if [ "${_variant}" = "-openssl" ]; then
-  _namesuff=
-else
-  _namesuff="${_variant}"
-fi
-
 _realname=curl
 pkgbase=mingw-w64-${_realname}
-pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}${_namesuff}"
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}"
+         "${MINGW_PACKAGE_PREFIX}-${_realname}-gnutls"
+         "${MINGW_PACKAGE_PREFIX}-${_realname}-winssl")
```

The gnutls issue is under active investigation.
> Is it possible to opt out some of the sub-packages on clang?

```suggestion
pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}"
         $( [[ ${MINGW_PACKAGE_PREFIX} == *-clang-* ]] || echo "${MINGW_PACKAGE_PREFIX}-${_realname}-gnutls")
         "${MINGW_PACKAGE_PREFIX}-${_realname}-winssl")
```

untested...